src/hubbleds/pages/05-class-results-&-uncertainty/__init__.py

```diff
@@ -711,14 +723,12 @@ def update_class_slider_subset(id, highlighted):
             show=COMPONENT_STATE.value.is_current_step(Marker.con_int2c),
             event_fr_callback=lambda event: fr_callback(event=event, local_state=LOCAL_STATE),
             state_view={
-                "hint1_dialog": COMPONENT_STATE.value.age_calc_state.hint1_dialog,
-                "hint2_dialog": COMPONENT_STATE.value.age_calc_state.hint2_dialog,
-                "low_guess": get_free_response(LOCAL_STATE, "likely-low-age").get("response"),
-                "high_guess": get_free_response(LOCAL_STATE, "likely-high-age").get("response"),
-                "best_guess": get_free_response(LOCAL_STATE, "best-guess-age").get("response"),
+                "low_guess": get_free_response(LOCAL_STATE, "likely-low-age"),
+                "high_guess": get_free_response(LOCAL_STATE, "likely-high-age"),
+                "best_guess": get_free_response(LOCAL_STATE, "best-guess-age"),
```

Comment:
I think we do need to get the response here, since on this guideline I'm seeing what's below

![Screenshot from 2024-07-11 11-10-44](https://github.com/cosmicds/hubbleds/assets/14281631/5fa9841b-0cac-4834-83dd-9d0ae1e409dc)